plugins/modules/azure_rm_containerregistry_tag_info.py

```diff
@@ -0,0 +1,232 @@
+#!/usr/bin/python
+#
+# Copyright (c) 2022 Ross Bender (@l3ender)
+#
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+
+DOCUMENTATION = '''
+---
+module: azure_rm_containerregistry_tag_info
+version_added: "1.12.0"
+short_description: Get Azure Container Registry tag facts
+description:
+    - Get facts for Container Registry tags.
+
+options:
+    registry:
+        description:
+            - The name of the container registry.
+        type: str
+        required: true
+    repository_name:
+        description:
+            - Filter results for repository within the registry. If omitted, all repositories will be retrieved.
+        type: str
+    name:
+        description:
+            - Filter results by tags with a desired name.
+        type: str
+
+extends_documentation_fragment:
+    - azure.azcollection.azure
+    - azure.azcollection.azure_tags
+
+author:
+    - Ross Bender (@l3ender)
+'''
+
+EXAMPLES = '''
+- name: Get tags for all repositories in registry
+  azure_rm_containerregistry_tag_info:
+    registry: myRegistry
+
+- name: List tags for a specific repository
+  azure_rm_containerregistry_tag_info:
+    registry: myRegistry
+    repository_name: myRepository
+
+- name: List tags matching a name for a specific repository
+  azure_rm_containerregistry_tag_info:
+    registry: myRegistry
+    repository_name: myRepository
+    name: myTag
+'''
+
+RETURN = '''
+repositories:
+    description:
+        - A list of dictionaries containing facts for repositories.
+    returned: always
+    type: complex
+    contains:
+        name:
+            description:
+                - The name of the repository.
+            returned: always
+            type: str
+            sample: my-app
+        tags:
+            description:
+                - A list of dictionaries for the tags in the repository.
+            returned: always
+            type: list
+            suboptions:
+                name:
+                    description:
+                        - Name of the tag.
+                    type: str
```

Comment:
Missing "returned", Please add "returned" to all return value!!
```suggestion
                    type: str
                    returned: always
```